frontend/pages/apps/collection/projectgroup/[projectgroup].tsx

```diff
@@ -0,0 +1,48 @@
+import { GetStaticPaths, GetStaticProps } from "next"
```

Comment:
would rename the url to /group/ instead of /projectgroup/

Reply:
isn't the important part of info the `project` part? This endpoint basically shows that something belongs to a project. Groups are potentially very different things (and might infact be something we might want to use later for something else)

Reply:
It all depends on how this is represented in the UI side. But yeah, at least make it `project-group`. `projectgroup` is a bit too much

Reply:
I personally find projectgroup without a - in the URL nicer, but I want to hear what others think.

Reply:
I don't really care, if we're only talking about the url, tbh. It would probably be a bit more consistent, if we do `project-group`, as the spec has a break (underscore) there too https://www.freedesktop.org/software/appstream/docs/chap-CollectionData.html#tag-ct-project_group

Reply:
I have now renamed the URL